sdks/python/apache_beam/yaml/main.py

```diff
@@ -67,6 +105,12 @@ def _pipeline_spec_from_args(known_args):
 def run(argv=None):
   known_args, pipeline_args = _configure_parser(argv)
   pipeline_yaml = _pipeline_spec_from_args(known_args)
+  jinja_variables = _extract_jinja_variables(known_args, pipeline_args)
+  if (known_args.jinja_variables is not None or
+      known_args.flags_as_jinja_variables):
+    pipeline_yaml = jinja2.Environment(
+        undefined=jinja2.StrictUndefined).from_string(pipeline_yaml).render(
+            **jinja_variables)
```

Comment:
Could we add a test or 2 to show this works, both with pipeline options and `--jinja_variables`?

Reply:
I'll do that.

Reply:
Done.